homepage/homepage.css

```diff
@@ -0,0 +1,252 @@
+@import url(../nav1.css);
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.gallery{
+    margin: 10px auto;
+    display:grid;
+    grid-template-columns: repeat(3,1fr);
+    column-gap:5px;
```

Comment:
Instead of row-gap and column-gap , you can use the shorthand i.e gap property itself